felix/rules/endpoints.go

```diff
@@ -561,17 +578,18 @@ func (r *DefaultRuleRenderer) endpointIptablesChain(
 					//
 					// For untracked and pre-DNAT rules, we don't do that because there may be
 					// normal rules still to be applied to the packet in the filter table.
-					rules = append(rules, generictables.Rule{
-						Match:  r.NewMatch().MarkClear(r.MarkPass),
-						Action: r.Nflog(nflogGroup, CalculateEndOfTierDropNFLOGPrefixStr(dir, tier.Name), 0),
-					})
-
+					if flowLogEnabled {
```

Comment:
in the end of the day, it seems like this is the only place where you need it. It does not need to be plumbed through all the func calls :thinking: